contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -100,7 +104,7 @@ The user experience for joining a cluster would be something like:
 kubeadm join --token=ae23dc.faddc87f5a5ab458 <address>
 ```
 
-**Note:** This is logically a different use of the token from TLS bootstrap.  We harmonize these usages and allow the same token to play double duty.
+**Note:** This is logically a different use of the token used for authentication for TLS bootstrap.  We harmonize these usages and allow the same token to play double duty.
 
 #### Implementation Flow
 
```

Comment:
Yes -- approved and in the merge queue.  See https://github.com/kubernetes/kubernetes/pull/41281